tests/testthat/test-formatters.R

```diff
@@ -537,3 +537,15 @@ mpf <- MatrixPrintForm(strings = strs, spans = spans, aligns = aligns,
 
 expect_equal(length(grep("spn_val", toString(mpf))),
              1L)
+
+testthat::test_that("fmt_config works as expected", {
+  x <- fmt_config()
+  expect_identical(obj_format(x), NULL)
```

Comment:
Still need to figure out why this returns `\001NULL\001` instead of `NULL` (`x@format` returns correct result)